app/task_routes.py

```diff
@@ -0,0 +1,140 @@
+from app import db
+from flask import Blueprint
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import datetime
+# import requests
+import os
+from dotenv import load_dotenv
+from app.slack_api import notify_slack
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+load_dotenv()
+
+def validate_task(task_id):
```

Comment:
The `validate_task` function here is very similar to the `validate_goal` function in the goal routes, so you could extract this logic into a single helper function!